### PR TITLE
Migrate to Grabl

### PR DESCRIPTION
**Pull request by vmax**
## What is the goal of this PR?

Integrate building `kglib` with Grabl

## What are the changes implemented in this PR?

Add `.grabl/automation.yml` and remove old CircleCI config

### CODE REVIEW

File: .grabl/automation.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+      script: bazel test //:release-validate-deps --test_output=streamed
+  deployment:
+    deploy-github:
+      machine: graknlabs-ubuntu-20.04
+      script: |
+       wget https://github.com/tcnksm/ghr/releases/download/v0.12.0/ghr_v0.12.0_linux_386.tar.gz
+       tar xvf ghr_v0.12.0_linux_386.tar.gz
+       export VERSION_TAG=$(cat "VERSION")
+       curl -X POST --fail --data "{\"tag_name\": \"${VERSION_TAG}\",\"target_commitish\": \"${CIRCLE_SHA1}\",\"name\": \"KGLIB ${VERSION_TAG}\",\"body\": \"\",\"draft\": true,\"prerelease\": false}" https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/releases?access_token=${GITHUB_DEPLOYMENT_TOKEN}
+    deploy-maven-release:
```
Comment (jmsfltchr) [inline .grabl/automation.yml:103]: Should be `deploy-pypi`release

File: .grabl/automation.yml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      dependencies: [deploy-pip-snapshot]
+      machine: graknlabs-ubuntu-20.04
+      type: foreground
+      script: |
+        pyenv global 3.6.10
+        sudo unlink /usr/bin/python3
+        sudo ln -s $(which python3) /usr/bin/python3
+        sudo ln -s /usr/share/pyshared/lsb_release.py /opt/pyenv/versions/3.6.10/lib/python3.6/site-packages/lsb_release.py
+        bazel run //:grakn-extractor -- dist/grakn-core-all-linux
+        nohup ./dist/grakn-core-all-linux/grakn server start
```
Comment (jmsfltchr) [inline .grabl/automation.yml:70]: Ooh nice, running grakn here in the same job. Does this work actually? I think as of 2.0 Grakn will run as a foreground process, and so it'll need it's own background job in CI.

**Review by jmsfltchr — COMMENTED** [review]
Thanks for doing this update!! One correction, one discussion point to complete the review.

File: .grabl/automation.yml (hunk 1 above)
Comment (vmax) [reply .grabl/automation.yml]: @jmsfltchr fixed in https://github.com/graknlabs/kglib/pull/151/commits/8b04ca7a9d2daf75cfc411ed0d509e2f7dee1e8c

File: .grabl/automation.yml (hunk 2 above)
Comment (vmax) [reply .grabl/automation.yml:70]: @jmsfltchr I believe it does as `grakn server start` is currently a blocking command that starts Grakn. We can refactor it after we release & upgrade to 2.0

Comment (jmsfltchr) [reply .grabl/automation.yml:70]: Ok then!

**Review by jmsfltchr — APPROVED** [review]

**Pull request merged**